src/main/kotlin/com/cultureamp/kafka/connect/plugins/transforms/UnifyLegacySlackIntegrationPayload.kt

```diff
@@ -53,32 +66,46 @@ class UnifyLegacySlackIntegrationPayload<R : ConnectRecord<R>> : Transformation<
         return updatedValues
     }
 
-    private fun extractUnifiedValues(oauthResponseData: Struct): Triple<String, String, String> {
+    private fun extractUnifiedValues(oauthResponseData: Struct): Quintuple<String, String, String, String, String?> {
         var teamId: String
         var teamName: String
         var accessToken: String
+        var scope: String
```

Comment:
I saw in router, it's call `access_token_scopes`. We should align it

Reply:
Good call, will do.